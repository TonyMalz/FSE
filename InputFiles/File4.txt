1, ,U
2,1,I
3,2,1,2,3
4,2,5,6,7
5,1,U
6,5,1
7,5,7,8,9
