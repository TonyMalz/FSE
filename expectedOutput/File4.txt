1 : U -> {1, 7, 8, 9}
|
+- 2 : I -> {}
|  |
   +- 3 : {1, 2, 3} -> {1, 2, 3}
|  |
   +- 4 : {5, 6, 7} -> {5, 6, 7}
|
+- 5 : U -> {1, 7, 8, 9}
|  |
   +- 6 : {1} -> {1}
|  |
   +- 7 : {7, 8, 9} -> {7, 8, 9}